controllers/delayed.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+var redisModel = require('../models/redis'),
+    q = require('q');
+
+
+module.exports = function (app) {
+    var getDelayedModel = function(req, res){
+        var dfd = q.defer();
+        redisModel.getStatus("delayed").done(function(delayed){
```

Comment:
Ah or maybe we shouldn't catch here but let the routes catch and return an error response/render an error page.